api/src/main/java/com/github/commonsrdf/api/BlankNode.java

```diff
@@ -13,29 +13,43 @@
  */
 package com.github.commonsrdf.api;
 
+import java.util.UUID;
+
 /**
  * A <a href= "http://www.w3.org/TR/rdf11-concepts/#dfn-blank-node" >RDF-1.1
  * Blank Node</a>, as defined by <a href=
  * "http://www.w3.org/TR/rdf11-concepts/#section-blank-nodes" >RDF-1.1 Concepts
  * and Abstract Syntax</a>, a W3C Recommendation published on 25 February 2014.<br>
  * <p>
- * Note that: Blank nodes are disjoint from IRIs and literals. Otherwise, the
- * set of possible blank nodes is arbitrary. RDF makes no reference to any
- * internal structure of blank nodes.
+ * Note: 
+ * <blockquote cite="http://www.w3.org/TR/rdf11-concepts/#dfn-blank-node">
```

Comment:
Using blockquote cite="http://...." seems to cause issues for me with Java-8 Javadoc.